src/Liquifier.sol

```diff
@@ -204,6 +205,15 @@ contract Liquifier is Initializable, UUPSUpgradeable, OwnableUpgradeable, Pausab
         return eEthShare;
     }
 
+    // Swap user's eETH for Liquifier's stETH
+    function swapEEthForStEth(uint256 _amount) external whenNotPaused nonReentrant {
```

Comment:
just something to consider that this function might be heavily used by MEV bots in a event where we depeg. 

Bots can buy cheap eETH from a dex and swap it for stETH one to one here! Probably a good thing as this will help us repeg.

Reply:
yes! should we take fee

Reply:
No fees. I think this function can be very useful to symbiotic vault since they get a lot of eeth, but they are required to swap eeth to steth since symbiotic doesn't take eeth. With a fee they will continue to use the same method of getting eth and then minting steth which prevents a major use case which is helpful to us since our liquidity pool doesn't get drained. 

Maybe we should do a check if lido's token is worth less than our then it reverts so bots don't take advantage (like what dave recommended)

Reply:
makes sense!